examples/multi_text.rs

```diff
@@ -0,0 +1,71 @@
+use fm_index::converter::IdConverter;
+use fm_index::{Match, MatchWithTextId, MultiTextFMIndexWithLocate, Search};
+
+fn main() {
+    // When using MultiTextFMIndex, the text is concatenated with an end marker \0.
+    let text = concat!(
+        // 1
```

Comment:
We may later add a builder to feed MultiTextFMIndex with multiple texts concatenated with `\0` like `TextBuilder` in #33.